### ISSUE TITLE
NVDA freezes when a progress dialog from Audacity is present

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3316]
**Reported by k_kolev1985 on 2013-07-01 11:19**
Since I've started using Audacity, I've noticed that when a progress indication dialog is present from it, nVDA will freeze until the dialog goes away.

Steps to reproduce it.
1. Open Audacity.
2. Open an audio file in Audacity (the bigger the file - the better). A progress indication dialog appears, witch displays the progress of the open file operation.
3. Save the file in another (eg. mp3) file, from "File -> Export" command. During the export operation, another progress indication dialog appears.
Expected result: NVDA should be responsive and be able to read the screen, including those progress indication dialogs.
Actual result: NVDA freezes and doesn't read anything in any window (application) opened, until the current progress indication dialog disappears.

Notes:
1. NVDA version used: next-9303,7db4c21 (in bulgarian). Audacity version used: 2.0.3 (in bulgarian). Windows version used: Windows 7 Ultimate SP1 32-bit (in bulgarian and with bulgarian locale settings).
2. Hardware: Desktop PC; Processor Intel(R) Core(TM)2 Duo CPU E7400 @2.80GHz (2.80 GHz); RAM: 4GB (3,25GB usable); Soundcard: Realtek HD Audio (ALC888).

I don't think hardware specs or CPU load are the main factor here, because I've seen another screen reader (JAWS) to read those dialogs and not freeze on a PC with much lower specs.

**Comment by ehollig** [comment]
It would be interesting to know if this works with the latest version of Audacity. CC @derekriemer Regardless, https://github.com/nvaccess/nvda/issues/3316#issuecomment-155307908 may be a good reason to close this.

**Comment by derekriemer** [comment]
This still can happen, but much less.

**Comment by Adriani90** [comment]
@Robert-J-H, your thoughts on this are very appreciated.
I think it is rather related to the power of the processor rather than NVDA itself. Audacity needs lot of ressources when complex tasks are being processed.
The same behavior can be seen when running an anti-virus scan with Avast or Malwarebytes.

**Comment by Adriani90** [comment]
@k_kolev1985 can you still reproduce this in Audacity?

**Comment by josephsl** [comment]
Hi, specifically with NVDA 2018.4 beta 4? If the issue is gone, I suggest closing this one, too, as this is an audio batching scenario covered by event tracking fix from @MichaelDCurran. Thanks.

**Comment by Robert-J-H** [comment]
I don't think that this issue is still valid. It refers to an old
Audacity, probably with WXWidgets 2.98.
There may be some new problems but not directly related to this one.
Therefore, please close it.

**Comment by k-kolev1985** [comment]
I haven't worked with Audacity recently and haven't noticed, but this issue seams to be really solved. So yes, it can be closed.

**Comment by ehollig** [comment]
Closing as worksforme
